test_libs/pyspec/eth2spec/utils/ssz/ssz_typing.py

```diff
@@ -393,11 +398,27 @@ def hash_tree_root(self):
         return hash_tree_root(self, self.__class__)
 
 
+class Bytes4(BytesN):
+    length = 4
+
+
+class Bytes32(BytesN):
+    length = 32
+
+
+class Bytes48(BytesN):
+    length = 48
+
+
+class Bytes96(BytesN):
+    length = 96
+
+
 # SSZ Defaults
 # -----------------------------
 def get_zero_value(typ):
```

Comment:
we'll get field/elem-type based coercion in 1180, so `get_zero_value` / `infer_type` doesn't have to be guessing types anymore. yay.